packages/manager/apps/public-cloud/src/app.module.js

```diff
@@ -92,6 +92,16 @@ angular
       );
     },
   )
+  .config(
+    /* @ngInject */ (ouiCalendarConfigurationProvider) => {
+      const lang = Environment.getUserLanguage();
+      return import(`flatpickr/dist/l10n/${lang}.js`)
+        .then((module) => {
+          ouiCalendarConfigurationProvider.setLocale(module.default[lang]);
```

Comment:
I wonder if this should belong to `@ovh-ux/manager-core` like other `oui` configurations (but as it is buit with rollup it will require to handle the lange import differently)

Reply:
Yes it could move to `@ovh-ux/manager-core` but it means that if we don't use `flatpickr` (= don't use `oui-calendar`) in an application we still should import `flatpickr`.